lib/_http_outgoing.js

```diff
@@ -354,6 +372,13 @@ function _writeRaw(data, encoding, callback) {
     encoding = null;
   }
 
+  const byteLen = Buffer.byteLength(data, encoding);
+  if (this[kSentContentLength] > 0 && this[kSentContentLength] !== NumberPOSITIVE_INFINITY) {
+    if (byteLen + this[kBytesWritten] > this[kSentContentLength])
+      throw new ERR_HTTP_CONTENT_LENGTH_EXCEEDED(byteLen + this[kBytesWritten], this[kSentContentLength]);
```

Comment:
It could also be shorter no?

Reply:
`this[kSentContentLength] > 0` is required to check if the user has manually set `content-length` or not. I added `this[kSentContentLength] !== NumberPOSITIVE_INFINITY` check in case the user sets the `content-length` to POSITIVE_INFINITY. After that its the main `content-length` check